app/components/ui/images.tsx

```diff
@@ -13,3 +13,113 @@ type ImageProps = ImgHTMLAttributes<HTMLImageElement> &
 export function Image({ ...props }: ImageProps) {
   return <UnpicImage {...props} />;
 }
+
+interface ImageDetails {
+  url: string;
+  altText: string;
+  title: string;
```

Comment:
do we need a required title field? wasn't able to see where it's used and felt a bit redundant from altText

Reply:
I've seen that the title field is primarily used to improve SEO ratings.

Reply:
Interesting! Do you have any blog posts or resources that you can share as a reference? Could be helpful to have this linked/documented in the PR description. Not a blocker at all, just curious!

Reply:
title will be optional!